src/components/Confirm/index.js

```diff
@@ -84,15 +84,16 @@ const Previews = ({capture, retakeAction, confirmAction} ) =>
 
 
 export default class Confirm extends Component {
-  completeCapture(method, side, capture, confirmCapture, nextStep) {
+  completeCapture(id, method, side, capture, confirmCapture, nextStep) {
+    capture.id = id
     confirmCapture({method, id: capture.id})
     confirmEvent(method, side)
     nextStep()
   }
 
   startApiUpload(method, side, capture, token, confirmCapture, nextStep, advancedValidation, onApiUpload, onApiError) {
     onApiUpload()
-    postToOnfido(capture, method, token, advancedValidation, () => this.completeCapture(method, side, capture, confirmCapture, nextStep), onApiError)
+    postToOnfido(capture, method, token, advancedValidation, ({id}) => this.completeCapture(id, method, side, capture, confirmCapture, nextStep), onApiError)
```

Comment:
Shame, I've grown to like implicit unpacking :)
I think it will make more sense to call it `apiResponse` because we are not returning JSON here but a JS object